mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/annotations/DaoProfile.java

```diff
@@ -26,18 +27,16 @@
  *
  * <p>Example:
  *
- * <p>*
- *
  * <pre>
  *  * &#64;Mapper
  *  * public interface InventoryMapper {
  *  *   ProductDao productDao(@DaoTable String executionProfile);
```

Comment:
Shouldn't this be `ProductDao productDao(@DaoProfile String executionProfile);`?

Reply:
👍 Good catch, will fix